decidim-core/app/controllers/concerns/decidim/force_authentication.rb

```diff
@@ -22,6 +22,7 @@ def ensure_authenticated!
       # Next stop: Let's check whether auth is ok
       unless user_signed_in?
         flash[:warning] = t("actions.login_before_access", scope: "decidim.core")
+        store_location_for(:user, request.path)
```

Comment:
I think you could just call [`store_current_location`](https://github.com/decidim/decidim/blob/c964194a0c8b79374f00bfdef1678c3457c8a523/decidim-core/app/controllers/decidim/application_controller.rb#L55) here, right?

Reply:
Yes, that's even better, my bad!